src/Html/Abstractions/build.cmd

```diff
@@ -0,0 +1,3 @@
+@ECHO OFF
```

Comment:
Minor but this and src/Html/Abstractions/build.sh are the only 2nd-level scripts where no 1st-level scripts exist outside src/Installers (where they might make sense &hellip; not sure). Suggest either moving the files up into src/Html/ or flattening src/Html/Abstractions/ down to src/Html.Abstractions/

Reply:
I've moved up this project to a single level. The `Installers` projects seem like a special case so I've let those be.